src/Form/formatters.js

```diff
@@ -155,6 +155,7 @@ const inferCountryCodeFromElement = (el) => {
  * @return {{expirationYear: string, expirationMonth: string}}
  */
 const getMMAndYYYYFromString = (expiration) => {
+    /** @type {string[]} */
```

Comment:
Not sure why, but ESLint was complaining about this.

Reply:
I'm  not getting any error. How are you running eslint?